src/cls/rgw/cls_rgw_types.h

```diff
@@ -97,12 +97,13 @@ struct rgw_bucket_dir_entry_meta {
   string content_type;
   uint64_t accounted_size;
   string user_data;
+  bool multipart;
```

Comment:
this shouldn't be called `multipart`, should be something like `accounted_entry`. Should default to `true`, and the logic should be flipped.